resources/dicts/thoughts/alive/general.json

```
@@ -1884,7 +1884,7 @@
         "thoughts": [
         "Recounts the gathering for r_c and {PRONOUN/r_c/poss} patients",
         "Brought r_c a berry to identify",
-        "Brought r_c an herb to identify"
+        "Brought r_c a herb to identify"
```

```suggestion
        "Brought r_c an herb to identify"
```
This should actually be "an" as "herb" is pronounced with a beginning vowel sound despite the first letter being a consonant.